build.gradle

```diff
@@ -28,6 +29,7 @@ allprojects {
     if (file("src/main/java").isDirectory()) {
         apply plugin: 'java'
     }
+    apply plugin: 'org.kordamp.markdown.convert'
```

Comment:
* `gradle/java.gradle` contains the list of plugins to be applied for all java projects. Could we apply the markdown plugin there ?

* Also, `gradle/*.gradle` contains the configuration used by each of the plugin e.g: `gradle/findbugs.gradle` contains the configuration for findbug plugin. Similarly can we have a `gradle/markdown.gradle` (We can choose a better name here)  file where the configuration for markdown plugin is configured?

Reply:
>gradle/java.gradle contains the list of plugins to be applied for all java projects. Could we apply the markdown plugin there ? 

We don't need this plugin for `.java` files. This is only needed for documents in `.md` format, which are present in pravega root folder.

>Also, gradle/*.gradle contains the configuration used by each of the plugin e.g: gradle/findbugs.gradle contains the configuration for findbug plugin. Similarly can we have a gradle/markdown.gradle (We can choose a better name here) file where the configuration for markdown plugin is configured? This will prevent build.gradle from bloating up.

I have created a`mkdocs.gradle` under the folder `gradle`. Unlike other plugins like `java`, (which has a specific config for tests ) `maven`( for publishing artifacts), there is nothing much to configure here. I have just added a condition to apply the plugins only for the root project.